config/locales/en.yml

```diff
@@ -1,5 +1,10 @@
 ---
 en:
+  account_confusion:
+    this_is_a_trial: |
+      <p class="govuk-body">We’re still building GOV.UK accounts.</p>
+      <p class="govuk-body">At the moment you can only use GOV.UK accounts to save your <a class="govuk-link" href=https://www.gov.uk/transition-check/questions">Brexit checker</a> results.</p>
```

Comment:
Broken link here:

`href=https://www.gov.uk/transition-check/questions"`
should be
`href="https://www.gov.uk/transition-check/questions"`